enhancements/network/static-ip-addresses-vsphere.md

```diff
@@ -0,0 +1,568 @@
+---
+title: static-ip-addresses-vsphere
+authors:
+  - rvanderp3
+reviewers: 
+  - JoelSpeed - machine API
+  - elmiko - machine API
+  - patrickdillon - installer
+  - jcpowermac - vSphere
+  - cybertron - networking
+  - zaneb - networking
+approvers:
+  - JoelSpeed
+  - patrickdillon
+  - cybertron
+api-approvers: 
+  - JoelSpeed
+creation-date: 2022-10-21
+last-updated: 2022-11-01
+tracking-link: 
+- https://issues.redhat.com/browse/OCPPLAN-9654
+see-also:
+  - /enhancements/network/baremetal-ipi-network-configuration.md
+  - /enhancements/installer/vsphere-ipi-zonal.md
+replaces:
+superseded-by:
+---
+
+# Static IP Addresses for vSphere IPI
+
+## Summary
+
+Static IP addresses are emerging as a common requirement in environments where
+the usage of DHCP violates corporate security guidelines.  Additionally, many 
+users which require static IPs also require the use of the IPI installer. 
+The proposal described in this enhacement discusses the implementation of
+of assiging static IPs at both day 0 and day 2.
+
+## Motivation
+
+Users of OpenShift would like the ability to provision vSphere IPI clusters with static IPs.
+
+- https://issues.redhat.com/browse/OCPPLAN-9654
+
+### User Stories
+
+As an OpenShift administrator, I want to provision nodes with static IP addresses so that I can comply with my organization's security requirements.
+
+As an OpenShift administrator, I want to provision static IP addresses with the IPI installer so that I can reduce the complexity of certifying tools required to provision OpenShift.
+
+As an OpenShift administrator, I want to scale nodes with static IPs so that I can meet capacity demands as well as respond to disaster recovery scenarios.
+
+As an OpenShift engineer, I want to know if a IPI installation was performed so that support can determine the IP assignment strategy for a given cluster.
```

Comment:
I don't quite follow this user story. Why does UPI vs IPI matter in the IP assignment strategy? How is UPI vs IPI install being determined by support?

Reply:
That's a fair point.  I'll think on this.  This was something the SPLAT team had discussed among ourselves early on.  I think we can strike this story.  I'll make that update in the morning.